Valigator.TestApi.Core31/Startup.cs

```diff
@@ -10,6 +13,31 @@
 [assembly: ApiController]
 namespace Valigator.TestApi
 {
+	internal class NullObjectModelValidator : IObjectModelValidator
```

Comment:
This stops the Asp.Net Core model validation from running when registered. This is essential to replicate the error I was having.